cartography/cli.py

```diff
@@ -531,6 +531,7 @@ def run_aws(request):
         neo4j_max_connection_lifetime=request['neo4j']['connection_lifetime'],
         credentials=request['credentials'],
         params=request['params'],
+        aws_requested_syncs=request.get('services', None),
```

Comment:
same as gcp, we should use the default services list, if `services` parameter is null.

Reply:
for this we already have logic in `__init.py__` file